### ISSUE TITLE
configurable i18n call expressions

### ISSUE THREAD

**Issue by Tinoooo** [intlify/eslint-plugin-vue-i18n#643]
**What rule do you want to change?**
@intlify/vue-i18n/no-unused-keys

**Does this change cause the rule to produce more or fewer warnings?**
fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
The rule currently checks for [a fixed set of call expressions](https://github.com/intlify/eslint-plugin-vue-i18n/blob/83dda8135589264afd7d08eebc2479cd291cd0a1/lib/utils/collect-keys.ts#L32). I propose to make additional expressions configurable.

**Please provide some example code that this change will affect:**

```vue
<script setup>
const { t } = useI18n()
const { t: tg } = useI18n({ useScope: 'global' })
</script>

<template>
component specific i18n message: {{ t('verySpecific') }} <br>
exists in global i18n message file: {{ tg('cancel') }}
</template>

<i18n lang="json" locale="en">
{
  "verySpecific": "Hey! I'm component specific."
}
</i18n>
```

**What does the rule currently do for this code?**
If does falsly reports about the "cancel" message in the global i18n file (en.json) being unused.

**What will the rule do after it's changed?**
It will not report incorrectly.

**Additional context**

**Comment by ota-meshi** [comment]
Thanks for posting the issue.

It's a bug, but as a workaround, I think it will work if you rewrite it like this:

```vue
<script setup>
// ...
const i18nGlobal = useI18n({ useScope: 'global' })
</script>

<template>
<!-- ... --> {{ i18nGlobal.t('cancel') }}
</template>
```

**Comment by Tinoooo** [comment]
Thank you for the workaround idea. I went wirh a pnpm patch for now.
